### ISSUE TITLE
NTR: Abdominal imaging: appendicitis (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9028]
New term request Abdominal imaging: appendicitis (seidel_190306194826): Ultrasound; CT; MRI; inflammation of the appendix; abdomen  UMLS: 0003615

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000143
- label: Appendicitis
- parents: [Abnormal large intestine morphology (HP:0002250)](https://hpo.jax.org/app/browse/term/HP:0002250)
- pmid: PMID:29630245
- def: Appendicitis is inflammation of the vermiform appendix. Appendix a hollow organ located at the tip of the cecum, usually in the right lower quadrant of the abdomen.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9028
- synonyms:
